example/topo/full_topo.go

```diff
@@ -24,17 +24,16 @@ func (t *FullTopology) GetLinkTypes() []string {
 	return []string{"Neighbors", "Master"}
 }
 
-func (t *FullTopology) GetNeighbors(linkType string, epoch uint64) []uint64 {
-	res := make([]uint64, 0)
-	switch {
-	case linkType == "Neighbors":
+func (t *FullTopology) GetNeighbors(linkType string, epoch uint64) (res []uint64) {
+	switch linkType {
+	case "Neighbors":
 		res = t.all
```

Comment:
The flagMeta's linktype should reverse.
Current flagmeta logic is a node send message as its own type
for example, a master send to slave, it need to invoke a flatmate as "master" role. 
It's wried compared to the comments.